compliance/tags/azure_rg_tags/azure_resource_group_tags.pt

```diff
@@ -172,36 +172,44 @@ end
 policy "policy_azure_resource" do
   validate_each $ds_filtered_resources do
     summary_template "{{ len data }} Azure Resources out of compliance."
-    detail_template <<-EOS
-# Azure Resources Incorrectly Tagged
-| Subscription Name | Resource Name | Resource Group | Azure Region | Resource ID | Tag Value | 
-| ------------- | -------------- | ------------ | ----------- | --------- | ------------ |
-{{ range data -}}
-| {{.subscriptionName}} | {{.name}} | {{.resource_group}} | {{.region}} | {{.id}} | {{.tag}}
-{{ end -}}
-
-###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }})
-EOS
     escalate $email
     escalate $tag_resources
     check eq(val(item, "resource_group"), val(item, "tag"))
+    export do
+      resource_level true
+      field "subscriptionName" do
+        label "Subscription Name" 
+      end
+      field "name" do
+        label "Resource Name"
+      end
+      field "resource_group" do
+        label "Resource Group"
+      end
+      field "region" do
+        label "Azure Region"
+      end
+      field "id" do
+        label "Resource ID"
+      end
+      field "tag" do
+        label "Tag Value"
```

Comment:
should be Tags, and the value should be the list of tag key:value pairs

Reply:
added the tags filed in the required format